mergify_engine/actions/merge/action.py

```diff
@@ -96,6 +96,8 @@ def _required_statuses_in_progress(ctxt, missing_conditions):
         need_look_at_checks = []
         for condition in missing_conditions:
             if condition.attribute_name.startswith("status-"):
+                # TODO(sileht): Just return True here, no need to checks
```

Comment:
Why not doing it now?
(hint: write the reason in this code comment rather than in a GitHub comment ;))